my-dsl.md

```diff
@@ -2,37 +2,85 @@
 _What is the name of the language? Link the name to its webpage 
 (if appropriate)._
 
+[Lout](http://www.adrianjwells.freeuk.com/lout.pdf) is a high-level markup language, similar to LaTex.
 
 # Domain
 _Describe the language's domain in five words._
 
+Typesetting documents, diagrams, and illustrations.
 
 # Computational model
 _What is the underlying computational model of this language? To answer this 
 question, provide a high-level description (no more than 100 words) of the 
 computation that occurs when someone executes a program in this language._
 
+Lout is a document formatter, so it reads a line of the Lout document and evaluates the expression(s) to generate a line of formatted text to add to the output document. It uses lazy evaluation to handle expressions, and can format computer program code in multiple languages, including C++, Java, Python, Perl, Ruby and Haskell.
 
 # DSL-ness
 _Fowler writes about a spectrum of languages, from general-purpose languages to 
 "purely" domain-specific. Where does the DSL you chose fall on this spectrum, 
 and why?_ 
 
+If document formatting is considered to be a tight enough domain, then Lout is very domain-specific, as it was designed solely for this purpose and has practically no use outside of it.
 
```

Comment:
I think layout / formatting is definitely a well-established domain. For example, the Markdown used on the assignment is another DSL in this domain.